src/widget/form/chatform.cpp

```diff
@@ -648,10 +559,9 @@ void ChatForm::onRejectCallTriggered()
     audioInputFlag = false;
     audioOutputFlag = false;
     emit rejectCall(callId);
-    micButton->setObjectName("green");
-    micButton->style()->polish(micButton);
-    volButton->setObjectName("green");
-    volButton->style()->polish(volButton);
+    
+    enableCallButtons();
+
 }
```

Comment:
There's a enableCallButtons here, but not in onHangupCallTriggered just above, is this intended?